detox/android/detox/src/full/java/com/wix/detox/espresso/matcher/ViewAtIndexMatcher.kt

```diff
@@ -21,6 +21,6 @@ class ViewAtIndexMatcher(private val index: Int, private val innerMatcher: Match
     }
 
     override fun describeTo(description: Description) {
-        description.appendText("matches " + index + "th view.")
+        description.appendText("View at index #$index, of those matching MATCHER$innerMatcher")
```

Comment:
Is it possible to include the total count here?

Reply:
Ah, I was opting do be able to do so, but unfortunately, it is not trivial to do with Espresso.
It's a shame because we'd end up for very similar message for iOS and Android.

Reply:
Sad. Okay. Nevermind.